docs/network-policy.md

```diff
@@ -60,61 +60,61 @@ spec:
 ```
 **OVN-Implementation:**
 
-Every new NetworkPolicy creates a port group named `FOO_bar` where `FOO` is the policy's Namespace and `bar` is the policy's name.  All pods that the policy's `podSelector` selects are added to the port group.
+Every new NetworkPolicy creates a port group named `<namespace>_<policyName>`.  All pods that the policy's `podSelector` selects are added to the port group.
 
-Additionally, two global deny PortGroups are also used, specifially: `IngressDefaultDeny` and `EgressDefaultDeny`.  Any pod selected by a NetworkPolicy in any Namespace is added to these PortGroups.
+Additionally, two deny PortGroups are created per-namespace, specifically: `<hashedNamespace>_IngressDefaultDeny` and `<hashedNamespace>_EgressDefaultDeny`.  Any pod selected by any NetworkPolicy in a Namespace is added to these PortGroups.
 
 subset of `ovn-nbctl find port-group`
 ```
-    _uuid               : 1deeac49-e87e-4e05-9324-beb8ef0dcef4
-    acls                : [3f864884-cdb7-44be-a60e-e4f743afc9d0, f174fcf1-a7c2-496d-9b96-136aaccc014f]
-    external_ids        : {name=ingressDefaultDeny}
-    name                : ingressDefaultDeny
-    ports               : [ce1bc4e5-0309-463f-9fd1-80f6e487e2d4]
-
-    _uuid               : 5249b7a2-36bf-4246-98ea-13d5c5e17c68
-    acls                : [36ed4154-71ab-4b8f-8119-5c4cc92708d9, c6663e29-99d0-4410-a2ff-f694a896a035]
-    external_ids        : {name=egressDefaultDeny}
-    name                : egressDefaultDeny
-    ports               : []
-
+_uuid               : bfb3a7a9-fb30-4b94-afc5-8c28345cb8f4
+acls                : [10f1baf9-2b4f-4947-9597-8e7159932977, 25614d28-0138-40c4-b331-2098a29042b9]
+external_ids        : {name=a16982411286042166782_egressDefaultDeny}
+name                : a16982411286042166782_egressDefaultDeny
+ports               : []
+
+_uuid               : dba665da-fb88-407c-837c-8b754ded2415
+acls                : [825f2270-3873-4478-aecb-8f28f49a6b92, 843fa1b7-8d59-4b2c-9a92-a4e7113492bf]
+external_ids        : {name=a16982411286042166782_ingressDefaultDeny}
+name                : a16982411286042166782_ingressDefaultDeny
+ports               : []
 ```
 
-Two ACLs (four total) are added to each PortGroup:
+Two ACLs (four total) are added to each DefaultDeny PortGroup:
 
-1. a drop policy with `priority=1000` and `direction=to-lport`
+1. a drop policy with `priority=9100`, `direction=to-lport`, and `name=<namespace>_<e/in>gressDefaultDeny`
```

Comment:
9100? Default deny priority is 1000.

Reply:
maybe this is after your refactor of ACLs or something? In which case we should hold this PR until that goes in?

Reply:
Yes exactly - I forgot to add the dependency. Let's wait for now